### PR TITLE
UT adjust test SPARK-26677: negated null-safe equality comparison

### PR DESCRIPTION
**Pull request by Feng-Jiang28**
This pr is for adjusting a test case that causes failure becuase it fetches the CPU version of Execution Plan instead of the GPU version.

### CODE REVIEW

**Comment by Feng-Jiang28** [comment]
build

**Review by thirtiseven — COMMENTED** [review]
Do we need update the status in `RapidsTestSettings` for this case?

**Comment by Feng-Jiang28** [comment]
build

File: tests/src/test/spark330/scala/org/apache/spark/sql/rapids/suites/RapidsParquetQuerySuite.scala [hunk 1]
```diff
@@ -19,9 +19,36 @@
...
 package org.apache.spark.sql.rapids.suites
 
+import com.nvidia.spark.rapids.GpuFilterExec
+
+import org.apache.spark.sql.DataFrame
 import org.apache.spark.sql.execution.datasources.parquet.ParquetQuerySuite
 import org.apache.spark.sql.rapids.utils.RapidsSQLTestsBaseTrait
 
 class RapidsParquetQuerySuite
   extends ParquetQuerySuite
```
Comment (thirtiseven) [inline tests/src/test/spark330/scala/org/apache/spark/sql/rapids/suites/RapidsParquetQuerySuite.scala:29]: nit: should the `extends` and `with` be in the same indent level?

**Review by thirtiseven — APPROVED** [review]

**Comment by Feng-Jiang28** [comment]
build

**Comment by Feng-Jiang28** [comment]
build

**Comment by Feng-Jiang28** [comment]
build

**Comment by Feng-Jiang28** [comment]
build

**Review by thirtiseven — APPROVED** [review]

**Pull request merged**